.github/scripts/windows/test_task.bat

```diff
@@ -54,7 +55,7 @@ set PDOTEST_DSN=odbc:%ODBC_TEST_DSN%
 
 rem setup Firebird related exts
 curl -sLo Firebird.zip https://github.com/FirebirdSQL/firebird/releases/download/v3.0.9/Firebird-3.0.9.33560-0_x64.zip
-7z x -oC:\Firebird Firebird.zip
+7z x -yoC:\Firebird Firebird.zip
```

Comment:
To be able to run multiple times, I just tell 7zip (undocumented dependency btw) to answer yes to all (overwrites existing)